tools/prlint/package.json

```diff
@@ -9,13 +9,9 @@
     "name": "Amazon Web Services",
     "url": "https://aws.amazon.com"
   },
-  "scripts": {
-    "build": "echo success"
-  },
   "license": "Apache-2.0",
   "dependencies": {
-    "github-api": "^3.3.0",
-    "make-runnable": "^1.3.6"
```

Comment:
were we not using it anywhere? I would have thought it's usage also needs to be updated?

Reply:
Yeah sorry, I removed the usage as well now. But it looks like the dependency upgrade caused some kind of error. Checking